pkg/providers/vsphere/network/network.go

```diff
@@ -660,22 +660,27 @@ func vpcSubnetPortToResult(
 	}
 
 	ipConfigs := []NetworkInterfaceIPConfig{}
+	ipAddresses := subnetPort.Status.NetworkInterfaceConfig.IPAddresses
+
+	// For now, we have an assumption that one subnetport only have one IP address.
```

Comment:
This code though doesn't really have this assumption, right?

Reply:
Nope.